contracts/Manager.sol

```diff
@@ -406,11 +423,56 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
             revert RebalanceEnoughCelo(toGroup, actualToCelo, expectedToCelo);
         }
 
-        scheduleRebalanceTransfer(
-            fromGroup,
-            toGroup,
-            Math.min(actualFromCelo - expectedFromCelo, expectedToCelo - actualToCelo)
+        uint256 receivableVotes = getReceivableVotesForGroup(toGroup);
+
+        if (receivableVotes == 0) {
+            revert ToGroupOverflowing(toGroup);
+        }
+
+        uint256 toMove = Math.min(
+            Math.min(actualFromCelo - expectedFromCelo, expectedToCelo - actualToCelo),
+            receivableVotes
         );
+
+        scheduleRebalanceTransfer(fromGroup, toGroup, toMove);
+    }
+
+    /**
+     * @notice Rebalance according to CELO overflow rather than stCELO ratio.
+     * If one of the groups is overflowing and there are still some votes that
+     * are scheduled for the group, this function allows to transfer these
+     * votes to any active group in protocol that is not overflowing yet.
```

Comment:
Should it go to any group, or specifically to the lowest voted group(s), as per the default strategy?